File: Can Santa save Christmas?.js
function determineTime (durations) {
  let total = 0
  for(let duration of durations){
    const [h,m,s] = duration.split(":").map(Number)
    total += (h*3600)+ (m*60)+s
  }
  const allowed_time  = 24*60*60
  return total <= allowed_time
}

File: Can Santa save Christmas?.py
def determine_time(durations):
    total = 0
    for duration in durations:
        h, m, s = map(int, duration.split(":"))
        total += (h * 3600) + (m * 60) + s
    allowed_time = 24 * 60 * 60
    return total <= allowed_time
